src/test/manifest-only-project.ts

```diff
@@ -71,6 +71,8 @@ describe('manifest project - answers & args', () => {
       helpers.run(path.join(__dirname, '../app')).withArguments(argument).withPrompts(answers).on('end', done);
     });
 
+    console.log("validating files")
```

Comment:
these console.log() calls were removed with the previous pull request. I think they are included accidentally here.